<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameFrame</class>
 <widget class="QDialog" name="GameFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Einser</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Zweier</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Dreier</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Vierer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Fünfer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Sechser</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Bonus ab 63</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>112</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Dreierpasch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Viererpasch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Full-House</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Kleine Straße</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Große Straße</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Kniffel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Chance</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>121</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Gesamt unten</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Gesamt oben</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Endsumme</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>81</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="button_einser">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_zweier">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_dreier">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_vierer">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_fuenfer">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_sechser">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>51</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="einser">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="zweier">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dreier">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="vierer">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fuenfer">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sechser">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="bonus">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>51</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="dreierpasch">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="viererpasch">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fullhouse">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="kleinestrasse">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="grossestrasse">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="kniffel">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="chance">
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>82</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QPushButton" name="button_drp">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonvip">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_fh">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_ks">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_gs">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_kniffel">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_chance">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>680</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="summe_unten">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="summe_oben">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="endsumme">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>20</width>
     <height>771</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="dice_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="dice_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="dice_1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="dice_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>170</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="dice_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="wuerfeln">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Würfeln</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
